shared/rakefiles/xk_util.rake

```diff
@@ -263,4 +263,19 @@ task :restore_snapshot_from_image_file, [:snapshot_files] => [@gcp_creds_file, :
   end
 
 end
+
+# This task forwards CouchDB port
+task :couchdb_ui => [:configure, :configure_secrets, :set_secrets] do
+  sh "sh -mc 'kubectl port-forward service/couchdb-svc-couchdb -n gpii --address=0.0.0.0 35984:5984 &
+    echo
+    echo \"CouchDB port is now being forwarded to your local machine, port 35984.\"
+    echo
+    echo \"You can access CouchDB Web UI at:\"
+    echo \"http://${TF_VAR_secret_couchdb_admin_username}:${TF_VAR_secret_couchdb_admin_password}@localhost:35984/_utils\"
```

Comment:
Don't like the idea of exposing plaintext credentials here. Maybe limit this task to only dev environments?

Reply:
We already have the `RAKE_REALLY_RUN_IN_PRD` protection, so I don't want to introduce another one really (also only admins would be able to run this in prod).

But I agree that exposing the credentials is not nice (and alternatives of copying those around do not seem better), and though if improving this story by generating short-lived credentials just for UI access, I'll look into this.